arduino/cores/packagemanager/install_uninstall.go

```diff
@@ -395,3 +411,27 @@ func (pme *Explorer) IsToolRequired(toolRelease *cores.ToolRelease) bool {
 	}
 	return false
 }
+
+// RunPostInstallScriptTool runs the post_install.sh (or post_install.bat) script for the
+// specified toolRelease.
+func (pme *Explorer) RunPostInstallScriptTool(toolRelease *cores.ToolRelease) error {
+	if !toolRelease.IsInstalled() {
+		return errors.New(tr("tool not installed"))
+	}
+	postInstallFilename := "post_install.sh"
+	if runtime.GOOS == "windows" {
+		postInstallFilename = "post_install.bat"
+	}
+	postInstall := toolRelease.InstallDir.Join(postInstallFilename)
+	if postInstall.Exist() && postInstall.IsNotDir() {
+		cmd, err := executils.NewProcessFromPath(pme.GetEnvVarsForSpawnedProcess(), postInstall)
+		if err != nil {
+			return err
+		}
+		cmd.SetDirFromPath(toolRelease.InstallDir)
+		if err := cmd.Run(); err != nil {
+			return err
+		}
+	}
+	return nil
+}
```

Comment:
This is mostly a copy of `RunPostInstallScript`, the only difference I see is the parameter (`*PlatformRelease` vs `*ToolRelease`). 
May you change the signature to `func (pme *Explorer) RunPostInstallScript(installDir *paths.Path) error` and keep just one function passing `platformRelease.InstallDir`/`toolRelease.InstallDir` instead of `platformRelease`/`toolRelease`.